wcfsetup/install/lang/de.xml

```diff
@@ -1019,7 +1019,7 @@ ACHTUNG: Die oben genannten Meldungen sind stark gekürzt. Sie können Details z
 		<item name="wcf.acp.language.item.languageCategoryID.mode.selection"><![CDATA[Manuelle Auswahl]]></item>
 		<item name="wcf.acp.language.item.languageCategoryID.description"><![CDATA[Der Text wird zur ausgewählten Text-Kategorie hinzugefügt.]]></item>
 		<item name="wcf.acp.language.item.languageItem"><![CDATA[Text-Bezeichner]]></item>
-		<item name="wcf.acp.language.item.languageItem.description"><![CDATA[Der Text-Bezeichner besteht aus mindestens drei durch Punkte getrennte Abschnitte. Kein Abschnitt darf leer sein und alle Abschnitten dürfen nur folgende Zeichen enthalten: <kbd>[A-z0-9-_]</kbd>.]]></item>
+		<item name="wcf.acp.language.item.languageItem.description"><![CDATA[Der Text-Bezeichner besteht aus mindestens drei durch Punkte getrennte Abschnitte. Kein Abschnitt darf leer sein und alle Abschnitte dürfen nur folgende Zeichen enthalten: <kbd>[A-z0-9-_]</kbd>.]]></item>
```

Comment:
```suggestion
		<item name="wcf.acp.language.item.languageItem.description"><![CDATA[Der Text-Bezeichner besteht aus mindestens drei durch Punkte getrennten Abschnitten. Abschnitte dürfen nicht leer sein und nur folgende Zeichen enthalten: <kbd>[A-z0-9-_]</kbd>.]]></item>
```
Same as above, also I've rephrased the second sentence entirely, because it read very strangely.